subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -164,16 +164,22 @@ uint8_t ll_scan_enable(uint8_t enable)
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
 	lll = &scan->lll;
 	if (duration) {
-		lll->duration_reload = ((uint32_t)duration * 10000U / 625U) /
+		lll->duration_reload = ((uint32_t)duration *
+					EXT_SCAN_DURATION_UNIT_US /
+					SCAN_INTERVAL_UNIT_US) /
```

Comment:
I wonder if it would be more readable to have a macro for converting this value, so it'd be `lll->duration_reload = CONVERT(duration, interval)` (with a much better name than `CONVERT` of course. Similar macro can be added for the duration calculation below.